### PR TITLE
Admin condition, completeness - support default values.

### PR DESCRIPTION
**Pull request by agorina**

### CODE REVIEW

File: model/src/main/java/org/opentestsystem/rdw/model/Opportunity.java [hunk 1]
```diff
@@ -886,58 +814,28 @@ public String getWindowOpportunity() {
...
-     *
-     * @param value
-     *     allowed object is
-     *     {@link String }
-     *
-     */
-    public void setCompleteStatus(String value) {
-        this.completeStatus = value;
-    }
-
```
Comment (agorina) [inline model/src/main/java/org/opentestsystem/rdw/model/Opportunity.java]: This is redundant.

File: model/src/main/java/org/opentestsystem/rdw/model/Opportunity.java [hunk 2]
```diff
@@ -201,16 +203,14 @@
...
- *
  */
 @XmlAccessorType(XmlAccessType.FIELD)
 @XmlType(name = "", propOrder = {
-    "segment",
-    "accommodation",
-    "score",
-    "genericVariable",
-    "item"
+        "segment",
```
Comment (malaffoon) [inline model/src/main/java/org/opentestsystem/rdw/model/Opportunity.java]: ?

File: model/src/main/java/org/opentestsystem/rdw/model/Opportunity.java [hunk 3]
```diff
@@ -313,25 +313,23 @@
 
     /**
      * Gets the value of the segment property.
-     *
+     * <p>
```
Comment (malaffoon) [inline model/src/main/java/org/opentestsystem/rdw/model/Opportunity.java]: ?
trivial edits to autogenerated comments seems like a waste

File: model/src/main/java/org/opentestsystem/rdw/model/Opportunity.java (hunk 2 above)
Comment (agorina) [inline model/src/main/java/org/opentestsystem/rdw/model/Opportunity.java]: For some reason my formatter decided to adjust it. Do you have a problem with it?

File: model/src/main/java/org/opentestsystem/rdw/model/Opportunity.java (hunk 3 above)
Comment (agorina) [inline model/src/main/java/org/opentestsystem/rdw/model/Opportunity.java]: OK, I will evert my re-formatting. Sorry about it- totally unintended.

File: build.gradle [hunk 4]
```diff
@@ -15,6 +15,7 @@ subprojects {
 
   dependencies {
     compile "org.slf4j:slf4j-api:1.7.22"
+    compile "com.google.guava:guava:21.0"
```
Comment (malaffoon) [inline build.gradle]: sigh; not sure it is worth bringing in guava just for this?

File: model/src/main/java/org/opentestsystem/rdw/model/Opportunity.java (hunk 1 above)
Comment (malaffoon) [inline model/src/main/java/org/opentestsystem/rdw/model/Opportunity.java]: But it is still part of the schema.

**Comment by agorina** [comment]
@malaffoon - rolled back the formatting changes.

File: model/src/main/java/org/opentestsystem/rdw/model/Opportunity.java (hunk 1 above)
Comment (agorina) [inline model/src/main/java/org/opentestsystem/rdw/model/Opportunity.java]: How do you suggest to handle it? Can we remove it from the schema?

File: model/src/main/java/org/opentestsystem/rdw/model/Opportunity.java [hunk 5]
```diff
@@ -2591,6 +2593,15 @@ private Score findOrCreateScore(final String of, final String label) {
         return score;
     }
 
+    /**
+     * A helper to return one of the two completeness options.
+     * @return
+     */
+    public String getBestCompleteness() {
+        return isNullOrEmpty(getCompleteness()) ?
+                ( isNullOrEmpty(getCompleteStatus()) ? "Complete" : getCompleteStatus() ): completeness;
```
Comment (malaffoon) [inline model/src/main/java/org/opentestsystem/rdw/model/Opportunity.java]: Okay, now you're starting to hate me, aren't you ...

This is kind of inconsistent. Personally i have no problem directly accessing the member variables:
```
    return isNullOrEmpty(completeness) ? isNullOrEmpty(completeStatus) ? "Complete" : completeStatus : completeness;
```

Comment (agorina) [inline model/src/main/java/org/opentestsystem/rdw/model/Opportunity.java]: Do you really have to be that difficult today? Ok, ok - changing once again.

**Comment by agorina** [comment]
@malaffoon, it is perfect now, I promise! :)

**Pull request merged**